Source/FieldSolver/WarpXPushFieldsHybridPIC.cpp

```diff
@@ -189,6 +189,18 @@ void WarpX::HybridPICEvolveFields ()
                            0, 0, 1, current_fp_temp[lev][idim]->nGrowVect());
         }
     }
+
+    // Check that the E-field does not have nan or inf values, otherwise print a clear message
+    ablastr::fields::MultiLevelVectorField Efield_fp = m_fields.get_mr_levels_alldirs(FieldType::Efield_fp, finest_level);
+    for (int lev = 0; lev <= finest_level; ++lev)
+    {
+        for (int idim = 0; idim < 3; ++idim) {
+            WARPX_ALWAYS_ASSERT_WITH_MESSAGE(
+                Efield_fp[lev][idim]->is_finite(),
```

Comment:
@WeiqunZhang this is the PR we mentioned in the developer meeting yesterday. 
I found that if I use `Efield_fp[lev][idim]->is_finite(true)` to avoid the MPI reduction the execution hangs (presumably due to only some processes returning `false`).
I've tested this by modifying the magnetic reconnection test to make it unstable. I attached the input script for the unstable case if you want to play around with it.

[warpx_inputs.txt](https://github.com/user-attachments/files/17651778/warpx_inputs.txt)